.github/workflows/build.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
       - uses: chainguard-dev/actions/goimports@main
       - run: |
           go build ./...
-          go test ./...
+          go test -race ./...
```

Comment:
can we add the `-v` as well just to make it verbose

Reply:
This PR was subsumed by another, so closing it out.